tendrl/ceph_integration/persistence/persister.py

```diff
@@ -1,11 +1,22 @@
 from tendrl.ceph_integration.persistence.sync_objects import SyncObject
+from tendrl.commons.config import load_config
+from tendrl.commons.etcdobj.etcdobj import Server as etcd_server
 from tendrl.commons.persistence.etcd_persister import EtcdPersister
 
+config = load_config(
+    "gluster-integration",
+    "/etc/tendrl/tendrl.conf"
```

Comment:
Might not work. tendrl.conf is gone and you will have module specific files. Also this function expected to load yaml files